app/models/checkout.rb

```diff
@@ -1,34 +1,43 @@
 class Checkout < ApplicationRecord
   CHECKOUT_PERIOD_IN_DAYS = 14
-  FINE_AMOUNT_PER_DAY_IN_CENTS = 0.10
+  FINE_AMOUNT_PER_DAY_IN_CENTS = 10
 
   belongs_to :user
   belongs_to :book
 
   validates :user, presence: true
-  validates :book, presence: true, uniqueness: true
+  validates :book, presence: true #, uniqueness: true
+  validates :due_on, presence: true
 
-  def due_date
-    self.created_at + CHECKOUT_PERIOD_IN_DAYS.days
-  end
+  monetize :fine_cents
 
-  def fine
```

Comment:
@paulcsmith, Should I abstract all methods related to fines to a `Fine model`? There are fine methods is `checkout.rb` and `user.rb`.